### PR TITLE
For Queen and Country

### PR DESCRIPTION
**Pull request by tomlarkworthy**
@sararob

### CODE REVIEW

File: translations.js [hunk 1]
```diff
@@ -1,46 +1,62 @@
 var translations = {
 	"0": {
-		"american": "a little bit",
-		"british": "a spot"
```
Comment (tomlarkworthy) [inline translations.js:4]: no-one actually uses that expression

File: translations.js [hunk 2]
```diff
@@ -1,46 +1,62 @@
...
+	"1": {
+		"american": "cookie",
+		"british": "biscuit"
+	},
 	"2": {
 		"american": "biscuit",
 		"british": "scone"
 	},
 	"3": {
-		"american": "shoes",
```
Comment (tomlarkworthy) [inline translations.js:15]: we have the word shoes

**Comment by sararob** [comment]
@tomlarkworthy looks awesome, thank you! :gb:

File: translations.js (hunk 1 above)
Comment (puf) [inline translations.js:4]: Sorry Tom, I was in the UK more recently than you. They all do! :-p

**Pull request merged**